source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -91,6 +116,19 @@ This section explains how to configure the supported IPAM options.
     created has to include the **other_config: ipv6_prefix** key defined.
     The value should be the the IP prefix featured in the provisioned cidr
     parameter, ending in '::' - e.g. 8230:5678::, for instance.
+* dhcpv6-stateless
+  - create a DHCP_Options similar to the one described above, also setting the
+    **options:dhcpv6** flag, and storing in its *external_ids* the
+    **gateway_ip** of the subnet. The former attribute indicates OVN central
+    to not include the IP address in the dhcpv6 packets - as per
+    the specification - whereas the latter attribute will be used to attach the
+    subnet to an OVN logical router.
+  - create an OVN logical router, indicating the subnet IDs it will connect.
+    For every subnet added, the IP address of its gateway will be extracted
+    from its external IDs, and used to correctly configure the router's
+    interfaces. An ansible based example can be found in the [ansible](#configuring-stateless-dhcpv6-on-the-subnet) section.
+    When using the REST interface directly, it is needed to first create the
+    router, and only then add the subnet interfaces to it, as per the [networking API](https://developer.openstack.org/api-ref/network/v2/?expanded=add-interface-to-router-detail#routers-routers).
```

Comment:
Why is that?
When creating a router, you need to specify a "subnet_id": 
so how i can create router before i have the subnet id? what i miss?

Reply:
You missed that the subnet is created before, in [0]; the subnet networking API object is encoded in an OVN dhcp_options object.

I removed plenty of this, since it probably was confusing.

Never the less, this section 'explains how to configure the supported IPAM options in OVN's data model'. Do you care about OVN's data model or plan on testing / manipulating it somehow?

[0] - https://github.com/maiqueb/ovirt-site/commit/95732338a527806b51bca0c7f2bf91987bc78460#diff-d58a2eb312890f2afd7a2b17f2463035R120